### PR TITLE
ADIOS2: Support NULL-Blocks (Chunks)

### PR DESCRIPTION
**Pull request by ax3l**
Add support to read back ADIOS2 data sets that contain empty blocks (NULL chunks). This is something to handle carefully and needed for a work-arounds in WarpX BTD.

X-Refs.:
- https://github.com/ornladios/ADIOS2/issues/3459
- https://github.com/ECP-WarpX/WarpX/pull/3657

### CODE REVIEW

**Review by RemiLehe — APPROVED** [review]

File: openpmd_viewer/openpmd_timeseries/data_reader/io_reader/utilities.py [hunk 1]
```diff
@@ -66,6 +66,15 @@ def get_data(series, record_component, i_slice=None, pos_slice=None,
         data = np.full(record_component.shape, np.nan, record_component.dtype)
         for chunk in chunks:
             chunk_slice = chunk_to_slice(chunk)
+
+            # skip empty slices
+            # https://github.com/ornladios/ADIOS2
```
Comment (ax3l) [inline openpmd_viewer/openpmd_timeseries/data_reader/io_reader/utilities.py:71]: Should read https://github.com/ornladios/ADIOS2/issues/3459

File: openpmd_viewer/openpmd_timeseries/data_reader/io_reader/utilities.py [hunk 2]
```diff
@@ -98,6 +107,15 @@ def get_data(series, record_component, i_slice=None, pos_slice=None,
             s_valid = list(s)  # same as s but reduced to valid regions in chunk
             s_target = []  # starts and stops in sliced array
             chunk_slice = chunk_to_slice(chunk)
+
+            # skip empty slices
+            # https://github.com/ornladios/ADIOS2
```
Comment (ax3l) [inline openpmd_viewer/openpmd_timeseries/data_reader/io_reader/utilities.py:112]: Should read https://github.com/ornladios/ADIOS2/issues/3459

File: openpmd_viewer/openpmd_timeseries/data_reader/io_reader/utilities.py (hunk 1 above)
Comment (ax3l) [reply openpmd_viewer/openpmd_timeseries/data_reader/io_reader/utilities.py:71]: #403

File: openpmd_viewer/openpmd_timeseries/data_reader/io_reader/utilities.py (hunk 2 above)
Comment (ax3l) [reply openpmd_viewer/openpmd_timeseries/data_reader/io_reader/utilities.py:112]: #403

**Pull request merged**